### PR TITLE
Update webhook-reference.md

### PR DESCRIPTION
**Pull request by JamieChapman8462**
Added levels for events and new warn event

## Description

Please describe what the changes are made in this pull request and why the change was necessary.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

**Comment by lornajane** [comment]
To be clear, will this change the format of existing webhooks? Tagging @mheap as I'm not sure if he will want to do an API review on a webhooks change (they're part of the API after all!) before we merge this.

**Comment by JamieChapman8462** [comment]
> To be clear, will this change the format of existing webhooks? Tagging @mheap as I'm not sure if he will want to do an API review on a webhooks change (they're part of the API after all!) before we merge this.

Yes this will change exiting events, but only the previous error ones.

One another notes this shouldn't be merge until the implementation work has finished.

File: _documentation/voice/voice-api/webhook-reference.md [hunk 1]
```diff
@@ -315,7 +315,19 @@ The event endpoint will also receive webhooks in the event of an error. This can
...
 `conversation_uuid` | `CON-aaaaaaaa-bbbb-cccc-dddd-0123456789ab` | The unique identifier for this conversation
 `timestamp` | `2020-01-01T12:00:00.000Z` | Timestamp (ISO 8601 format)
 
+## Warns
+
+The event endpoint will also receive webhooks in the event of an warning. This can be useful when debugging your application.
+
+Field | Example | Description
+ -- | -- | --
+`level` | `warn` | `indicates that the call was able to continue but could have been a better user experiance`
```
Comment (tbedford) [inline _documentation/voice/voice-api/webhook-reference.md]: Should the third item in the first row be in backticks?

**Comment by mheap** [comment]
Let's get this on nexmo/api-standards and make sure that we're happy with the format before merging

**Comment by mheap** [comment]
Closing this as the problem was solved without the need for a webhook and the guidelines have been updated in the API Standards project

**Pull request closed without merging**